source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst

```diff
@@ -307,6 +307,28 @@ requirements shown in Table 1:
      - /
      - *Size depends upon use case/desired bundles.*
 
+Disk encryption
+---------------
+
+Disk encryption is supported for root and swap partitions on |CL| for
```

Comment:
Disk encryption is supported by use of LUKS for all partitions with the exception of 'boot' on |CL| for

Reply:
@mdhorn. OK. I will refine the description per your response.  Also, to view the images in-line, select the "View" button in the upper right. BTW: I will change Fig. 11; it needs to show "Encrpyt" checked.
Here's the direct link to view images in line.
https://github.com/clearlinux/clear-linux-documentation/blob/da0e559c04b804abc2295fc2b69966777cbd0bc2/source/clear-linux/get-started/bare-metal-install-beta/bare-metal-install-beta.rst#advanced-options